### PR TITLE
Add `yodoctor` for backward compatibility

### PR DESCRIPTION
**Pull request by LitoMore**
~~This reverts commit a312a493b5d5ea58756028eefada368a2dd94fcf.~~

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,6 @@ node_js:
   - 8
 before_script:
   - npm link
-  - yo-doctor
+  - yodoctor
```
Comment (mischah) [inline .travis.yml]:
```suggestion
  - yo-doctor
```

File: readme.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Use as part of [`yo`](https://github.com/yeoman/yo):
 $ yo doctor
 ```
 
-Can also be run with `yo-doctor` if installed globally.
+Can also be run with `yodoctor` if installed globally.
```
Comment (mischah) [inline readme.md]:
```suggestion
Can also be run with `yo-doctor` if installed globally.
```

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
 	"repository": "yeoman/doctor",
 	"main": "lib/index.js",
 	"bin": {
-		"yo-doctor": "lib/cli.js"
+		"yodoctor": "lib/cli.js"
```
Comment (mischah) [inline package.json]:
```suggestion
		"yodoctor": "lib/cli.js",
		"yo-doctor": "lib/cli.js"
```

**Comment by LitoMore** [comment]
@mischah Oh haha, I didn't realize that! Thank you for correcting this!

**Pull request merged**